### ISSUE TITLE
[Suggestion] CSVHelper.Escape() method should check for actual separator value 

### ISSUE THREAD

**Issue by ig-sinicyn** [dotnet/BenchmarkDotNet#294]
Current codebase, CsvHelper.cs
```cs

        private static readonly char[] forbiddenSymbols = { '\n', '\r', '"', ',' };

        public static string Escape(string value)
        {
            // RFC 4180:
            // 2.6: Fields containing line breaks (CRLF), double quotes, and commas should be enclosed in double-quotes.
            // 2.7: If double-quotes are used to enclose fields, then a double-quote appearing inside a field must be escaped by preceding it with another double quote.
            if (forbiddenSymbols.Any(value.Contains))
                return Quote + value.Replace(Quote, TwoQuotes) + Quote;
            return value;
        }
```

Default exporter uses 
```cs
public static readonly IExporter Default = new CsvExporter(CsvSeparator.CurrentCulture);
```
and `CsvSeparator.CurrentCulture` is translated to `CultureInfo.CurrentCulture.TextInfo.ListSeparator`. For ru-RU culture the separator is `;`. As a result
```cs
        [Benchmark(Description = "Robert');DROP TABLE Students;--")]
        public void RunMe()
```
will result in broken .csv output.

**Comment by adamsitnik** [comment]
@alinasmirnova was this fixed with #408 ?

**Comment by alinasmirnova** [comment]
@adamsitnik not really. I've fixed it in #409
